cfgmgr/intfmgr.cpp

```diff
@@ -74,10 +101,12 @@ IntfMgr::IntfMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, c
     {
        mySwitchType = swtype;
     }
+
+    g_ipv6Flag = inferIPV6Capability();
```

Comment:
Why do we care about global configuration?

Reply:
Will remove